numpy/testing/_private/utils.py

```diff
@@ -35,8 +35,7 @@
         'assert_allclose', 'IgnoreException', 'clear_and_catch_warnings',
         'SkipTest', 'KnownFailureException', 'temppath', 'tempdir', 'IS_PYPY',
         'HAS_REFCOUNT', 'suppress_warnings', 'assert_array_compare',
-        '_assert_valid_refcount', '_gen_alignment_data', 'assert_no_gc_cycles',
-        'break_cycles', 'HAS_LAPACK64'
+        'assert_no_gc_cycles', 'break_cycles', 'HAS_LAPACK64'
```

Comment:
The private functions are actually used and imported directly from `numpy.testing`, which in turn imports them with `*`.

Reply:
IIRC, they are due to @seberg . I've wondered about their private status.

Reply:
Note that respective private functions are still imported explicitly to `np.testing.__init__` and `np.testing.utils`, 
so the content of these namespaces hasn't changed.
They're just no longer part of `__all__`, as putting referencing private objects within the `__all__` of a public module is not exactly great practice.